### PR TITLE
feat: merge panacea-core v2.0.5 into master

### PR DESCRIPTION
**Pull request by gyuguen**
## Comment
We have to redefine the v2.1.0-alpha2 version Upgradehandler.
So this upgradeHandler is deleted

### CODE REVIEW

File: .gitbook/guide/installation.md [hunk 1]
```diff
@@ -33,9 +33,12 @@ As a workaround, please add `replace` directives in your `go.mod` as below.
...
-go 1.17
+go 1.19
 
 replace (
+	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
+	github.com/cosmos/cosmos-sdk => github.com/medibloc/cosmos-sdk v0.45.9-panacea.1
+	github.com/cosmos/ledger-go => github.com/cosmos/ledger-go v0.9.3
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 	google.golang.org/grpc => google.golang.org/grpc v1.33.2
 	github.com/cosmos/cosmos-sdk => github.com/medibloc/cosmos-sdk v0.42.11-panacea.1
```
Comment (youngjoon-lee) [inline .gitbook/guide/installation.md:44]: We can delete this line :)
```
github.com/cosmos/cosmos-sdk => github.com/medibloc/cosmos-sdk v0.42.11-panacea.1
```

**Review by youngjoon-lee — APPROVED** [review]
so clean!

**Review by H4NLee — APPROVED** [review]
lgtm
BTW, `lint` ci failed because of the upgrade of testify version.
Don't we have to fix it as well in this PR?

**Comment by gyuguen** [comment]
> lgtm BTW, `lint` ci failed because of the upgrade of testify version. Don't we have to fix it as well in this PR?

Yes, I will fix as soon.

**Review by youngjoon-lee — APPROVED** [review]

**Pull request merged**